src/main/com/yetanalytics/persephone/pattern/fsm_spec.cljc

```diff
@@ -282,3 +282,26 @@
     (fn [] (sgen/fmap
             dfa-gen-fmap
             (s/gen :set-dfa/dfa-basics)))))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; State Info Specs
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(s/def ::accepted? boolean?)
+(s/def ::visited (s/coll-of ::symbol-id))
+
+(def state-info-spec
+  (s/every (s/keys :req-un [:int-dfa/state
+                            ::accepted?
+                            ::visited])
```

Comment:
this is `:req-un`, but isn't included unless asked for by the user

Reply:
https://github.com/yetanalytics/project-persephone/pull/27/commits/8e3e9f1ee28f7a6a7dd286d098f31c4428117219

Reply:
noice!